audit/README.md

```diff
@@ -78,8 +78,8 @@ future changes by simply running `terraform apply
 | Name | Source | Version |
 |------|--------|---------|
 | cw\_alarm\_sns | github.com/cisagov/sns-send-to-account-email-tf-module | n/a |
-| new\_user\_event | github.com/cisagov/new-user-alert-tf-module | n/a |
-| new\_user\_sns | github.com/cisagov/sns-send-to-account-email-tf-module | n/a |
+| user\_group\_mod\_event | github.com/cisagov/user-group-mod-alert-tf-module | n/a |
+| user\_group\_mod\_sns | github.com/cisagov/sns-send-to-account-email-tf-module | n/a |
 | provisionaccount | github.com/cisagov/provisionaccount-role-tf-module | n/a |
```

Comment:
These should be alphabetized.  The same goes for the other `README.md` files in this repo.

Reply:
Good catch!  Please see https://github.com/cisagov/cool-accounts/pull/140/commits/0ce4e5182517af04d781184bb95365532257b5ba.

Look for an upcoming PR shortly to add our standard `.terraform-docs.yml` file to each subdirectory, something that we overlooked when https://github.com/cisagov/skeleton-tf-module/pull/189 trickled down to this repo.

Reply:
FYI, PR #141 has been created.

Reply:
Thanks @dav3r!